packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -566,6 +566,17 @@ async function createUniswapPriceFeedForFinancialContract(
 
   const userConfig = config || {};
 
+  // Check if there is an override for the getTime method in the price feed config. Specifically, we can replace the
```

Comment:
this `if` logic before enabled the uniswap feeds to be used correctly in hard hat when created from the `createReferencePriceFeedForFinancialContract` method. I added this branching logic to the uniswap feeds as well as this kind of config is needed when testing the trader bot when using the `createTokenPriceFeedForFinancialContract` method. This only effects unit tests.